cassandra_backup_service.py

```diff
@@ -1602,9 +1602,10 @@ def status(self, columnfamily, restore_time):
 
         return backup_status
 
-    def restore(self, columnfamily, nodes, restore_time):
-        run_command(['rm', '-rf', '/tmp/restore'])
-        run_command(['rm', '-rf', '/tmp/restore-download'])
+    def restore(self, columnfamily, nodes, restore_time, restore_dir):
+        restore_dir = restore_dir.rstrip('/')
+        run_command(['rm', '-rf', restore_dir])
+        run_command(['rm', '-rf', '{0}/download'.format(restore_dir)])
```

Comment:
Doesn't the first one delete this directory too?

Reply:
It does, this is a remnant of when I had the `download` directory in a different location instead of contained within the temp directory. It seemed cumbersome to have the user provide a temp directory and a download directory and I just forgot to delete this line. Fixed: https://github.com/onzra/cassandra_backup_service/pull/8/commits/1fd4b12b46acf446c99bd791b12bca8dae0482e8